telegram/_inline/inlinekeyboardbutton.py

```diff
@@ -111,6 +112,10 @@ class InlineKeyboardButton(TelegramObject):
             in inline mode when they are currently in a private chat with it. Especially useful
             when combined with ``switch_pm*`` actions - in this case the user will be automatically
             returned to the chat they switched from, skipping the chat selection screen.
+
+            Tip:
+                This does the same as :paramref:`switch_inline_query_current_chat`, but for some
+                reason, Telegram has not deprecated this field.
```

Comment:
it does not. current chat explicitly switches to the current chat, `switch_inline_query` allows the user to choose *any* chat

Reply:
~darn copilot~, I meant the new parameter hehe